src/templates/storyViewer.handlebars

```diff
@@ -57,5 +57,12 @@
         <div hidden>
             <a href="#" class="sjrkc-st-story-save-no-share sjrk-st-story-save-no-share">{{localizedMessages.message_saveNoShareText}}</a>
         </div>
+        <div class="sjrkc-st-story-share-progress sjrk-st-story-share-progress" aria-live="assertive">
+            <div class="sjrk-st-story-share-progress-indicator" aria-labelledby="sjrkc-st-story-share-progress-text"></div>
```

Comment:
As discussed with @BlueSlug and @klown at fluid-work channel (the discussion can be found found in [this log](http://irc-logs.fluidproject.org/%23fluid-work/%23fluid-work.2019-02-15.log) from 2019-02-15T21:46:22), @BlueSlug will look into using [aria progressbar attritute](https://www.w3.org/TR/wai-aria-1.1/#progressbar) for this progress indicator.